src/java.desktop/share/classes/java/beans/beancontext/package-info.java

```diff
@@ -30,12 +30,15 @@
  * context can be nested within another bean context. This package also
  * contains events and listener interface for beans being added and removed from
  * a bean context.
- *
- * This package has been deprecated and may be removed in a future version of the Java Platform
- * There is no replacement.
- *
+ * <p>
+ * These APIs are now both obsolete and express an "anti-pattern" of component
```

Comment:
Why is “anti-pattern” in quotes? The word is used in its literal meaning, isn't it?

To emphasize, `<i>` or better `<em>` could be used.

Reply:
It's a copy from [the release notes](https://bugs.openjdk.org/browse/JDK-8336785), updated.